CodeStarterCamp_Week5/AuditionManager.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AuditionManager.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by Hyungmin Lee on 2023/03/20.
+//
+
+import Foundation
+
+struct AuditionManager {
+    let totalApplicantsList: [Person]
+    var passedApplicantsList = [Person]()
+    
+    mutating func cast() {
+        for applicant in totalApplicantsList{
+            if let applicant = applicant as? Talent, applicant.isLevelA() == true {
+                if applicant is TalentedPersonWithBadPersonality {
+                   continue
+                }
+                
+                passedApplicantsList.append(applicant as! Person)
+            }
```

Comment:
```suggestion
        for applicant in totalApplicantsList {
            if let applicant = applicant as? Talent, applicant.isLevelA() == true {
                if applicant is TalentedPersonWithBadPersonality {
                   continue
                }
                
                passedApplicantsList.append(applicant as! Person)
            }
```
- 띄어쓰기가 안 되어 있네요!
- 21번 라인에서 강제 타입 캐스팅을 하셨네요! 
2주차 `Optional` 학습자료에서 `강제 옵셔널 바인딩`을 지양했던 것처럼, `강제 타입 캐스팅`도 지양합니다!
왜 강제로 옵셔널 바인딩이나 타입 캐스팅을 하면 안 되는지 찾아보셔도 좋을 거 같아요! 
- `totalApplicantsList`가 `Person` 타입의 배열이기 때문에, `Person` 타입이 아닌 객체가 들어가면 오류가 납니다.
따라서 `Person` 타입은 따로 `타입캐스팅`이 불필요해보여요!

Reply:
넵! 강제 타입캐스팅을 불필요하게 하지 않는 방향으로 수정해둘게요!